### DISCUSSION TITLE
Welcome to hybrid-issues Discussions!

### DISCUSSION THREAD

**Discussion by jdinan** [mpiwg-hybrid/hybrid-issues#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bkmgit** [comment]
Name: Benson Muite
Brief bio: Software developer interested in performance portability. Interested in having good examples in the MPI standard.

**Reply by jdinan** [reply]
Thanks for introducing yourself! If you have ideas for how we can improve the MPI standard, we would love to hear them. You can propose ideas here, or join our bi-weekly meetings (details are on the wiki).
